huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_kafka_consumer_group_test.go

```diff
@@ -0,0 +1,139 @@
+package dms
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getDmsKafkaConsumerGroupResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+	// getKafkaConsumerGroup: query DMS kafka consumer group
+	var (
+		getKafkaConsumerGroupHttpUrl = "v2/{project_id}/instances/{instance_id}/groups/{group}"
+		getKafkaConsumerGroupProduct = "dmsv2"
+	)
+	getKafkaConsumerGroupClient, err := cfg.NewServiceClient(getKafkaConsumerGroupProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating DmsKafkaConsumerGroup Client: %s", err)
+	}
+
+	// Split instance_id and group from resource id
+	parts := strings.SplitN(state.Primary.ID, "/", 2)
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("invalid id format, must be <instance_id>/<consumerGroup>")
+	}
+	instanceID := parts[0]
+	name := parts[1]
+	getKafkaConsumerGroupPath := getKafkaConsumerGroupClient.Endpoint + getKafkaConsumerGroupHttpUrl
+	getKafkaConsumerGroupPath = strings.ReplaceAll(getKafkaConsumerGroupPath, "{project_id}",
+		getKafkaConsumerGroupClient.ProjectID)
+	getKafkaConsumerGroupPath = strings.ReplaceAll(getKafkaConsumerGroupPath, "{instance_id}", instanceID)
+	getKafkaConsumerGroupPath = strings.ReplaceAll(getKafkaConsumerGroupPath, "{group}", name)
+
+	getKafkaConsumerGroupOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+	}
+	getKafkaConsumerGroupResp, err := getKafkaConsumerGroupClient.Request("GET", getKafkaConsumerGroupPath,
+		&getKafkaConsumerGroupOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving DmsKafkaConsumerGroup: %s", err)
+	}
+
+	getKafkaConsumerGroupRespBody, err := utils.FlattenResponse(getKafkaConsumerGroupResp)
+	if err != nil {
+		return nil, err
+	}
+	groupJson := utils.PathSearch("group", getKafkaConsumerGroupRespBody, nil)
+	groupState := utils.PathSearch("state", groupJson, nil)
+	if groupState == "DEAD" {
+		return nil, fmt.Errorf("error retrieving DmsKafkaConsumerGroup")
```

Comment:
```suggestion
		return nil, fmt.Errorf("error retrieving DMS Kafka consumer group")
```
and fix other places

Reply:
done